### ISSUE TITLE
Suppress notifications from Datadog synthetic monitors during deployments

### ISSUE THREAD

**Issue by acrollet** [department-of-veterans-affairs/va.gov-cms#6201]
## Description

As a CMS Devops engineer, I wish to not be interrupted by notifications during deployment windows, so that I can maintain focus and reduce notification fatigue.

## Implementation

It's possible that this may be able to be done within Pagerduty's config. For Datadog, the only option in their docs to do conditional assertions is with regex conditions: https://docs.datadoghq.com/synthetics/guide/or-logic-api-tests-assertions/

## Acceptance Criteria
- [ ] Pagerduty notifications are not sent when monitors fail during deployment windows

**Comment by ElijahLynn** [comment]
I'm thinking we can "pause/unpause" the tests that depend on deployments with the Datadog API in pre/post deploy tasks: 
![image](https://user-images.githubusercontent.com/1504756/130300225-5725b89c-aba8-47ce-9913-c8db7fc43cca.png)

But, there is a "Mute monitors by tag" too, which is may be what we want > https://www.datadoghq.com/blog/mute-datadog-alerts-planned-downtime/.  We could do it during a time window but we do out-of-band deployments so that wouldn't work so I think we should do it in the pre-deploy steps as an API call (https://docs.datadoghq.com/api/latest/downtimes/) and pass a start and expected end time to it, which is what we do with PagerDuty as well.

**Comment by acrollet** [comment]
@olivereri is this now obviated by the approach you figured out for the graphql synthetic?

**Comment by olivereri** [comment]
@acrollet 
> @olivereri is this now obviated by the approach you figured out for the graphql synthetic (in #6290)?

No, the solution in #6290 is pretty specific to that monitor. We'd want a dynamic way to set DataDog in maintenance mode. However, setting a scheduled maintenance window around our deploy time will get us most of the way there.
